models/base/sources.yml

```diff
@@ -12,3 +12,11 @@ sources:
           - name: event_id
           - name: emission_time
           - name: event
+  - name: reporting
+    database: "reporting"
+    tables:
+      - name: xapi_events_all
+        columns:
+          - name: event_id
+          - name: emission_time
+          - name: event
```

Comment:
why is this model added here?

I think you need to add the environment variables for the databases to the CI workflow:
```yaml
ASPECTS_EVENT_SINK_DATABASE: event_sink
ASPECTS_XAPI_DATABASE: xapi
DBT_PROFILE_TARGET_DATABASE: reporting 
```